app/src/full/java/io/homeassistant/companion/android/sensors/HealthConnectSensorManager.kt

```diff
@@ -438,15 +427,13 @@ class HealthConnectSensorManager : SensorManager {
     }
 
     override fun hasSensor(context: Context): Boolean {
-        return SDK_INT >= Build.VERSION_CODES.UPSIDE_DOWN_CAKE &&
+        return SDK_INT >= Build.VERSION_CODES.P &&
             HealthConnectClient.getSdkStatus(context) == HealthConnectClient.SDK_AVAILABLE
```

Comment:
The client will handle SDK checks for us

```suggestion
        return HealthConnectClient.getSdkStatus(context) == HealthConnectClient.SDK_AVAILABLE
```